### PR TITLE
Add automatic CI versioning

### PR DESCRIPTION
**Pull request by mdgbayly**
Updates travis.yml to use frauci-update-version to bump package.json version and tag commit on each commit/pr merge to master. Use travis releases provider to create a new github release matching tag.

### CODE REVIEW

**Comment by awikkerink** [comment]
The CI appears to be failing because you are on a fork. I'll push to a local branch to get the CI working.

File: package.json [hunk 1]
```diff
@@ -21,5 +21,5 @@
   "dependencies": {
     "frau-jwt": "^2.0.0"
   },
-  "version": "2.0.3"
+  "version": "4.0.4"
```
Comment (awikkerink) [inline package.json]: Did you mean to do 2 major version bumps? Is this to make the major version match?

Comment (mdgbayly) [reply package.json]: @awikkerink Good catch. No I did not mean to do another major version bump there. I must have typed the version incorrectly when I was updating my script. I'll put it back to 2.0.5.

Comment (mdgbayly) [reply package.json]: @awikkerink Actually just updated to 2.0.4 as I hadn't use that version. That would be awesome if you could try on a local branch for me to get the CI working.

**Comment by mdgbayly** [comment]
@awikkerink Just realized I don't have write permissions on this repo. Would you be able to push the merge button for me. Thanks.

**Pull request closed without merging**